lib/clients/reports.ex

```diff
@@ -68,4 +68,84 @@ defmodule GlassFactoryApi.Clients.Reports do
       {:error, message} -> raise message
     end
   end
+
+  @doc """
+  List rates and costs reports from the given client id
+
+  ## Examples
+
+      iex> GlassFactoryApi.Clients.Reports.list_rates_and_costs_reports(1234, [], config)
+      {:ok, [%RatesAndCostsReport{
+          client_id: 1234,
+          project_id: 12345,
+          job_id: nil,
+          activity_id: 123_456,
+          user_id: 222,
+          role_id: 1234,
+          date: "2018-06-19",
+          time: 8,
+          rate: 35,
+          cost: 35
+        }]
+      }
+  """
+  @spec list_rates_and_costs_reports(integer(), Keyword.t(), map) ::
+          {:error, any} | {:ok, [RatesAndCostsReport]}
+  def list_rates_and_costs_reports(client_id, opts \\ [], config \\ []) do
```

Comment:
Should `config` be a list or a map here?

Reply:
It's a map, sorry 😅